lesson8-UnitTesting/RecentlyUsedList/src/RecentlyUsedList.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace RecentlyUsedList
+{
+    public class RecentlyUsedListSolver
```

Comment:
Looking at the task, it asks for custom implementation of Stack. So consider renaming this class accordingly and instead of `Add` please introduce `Push` and `Pop` methods.

Reply:
done